test/fixtures/remote-error-test-cases/get-jstp-array.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
```

Comment:
I don't actually see any reason for the whole `fixtures/remote-error-test-cases` directory to exist. These files are not pure data, they contain quite a bit of imperative logic, and they are very small and isolated. I'd say their contents should be simply moved to the corresponding tests.